frontend/src/lib/__tests__/telemetry.test.ts

```
@@ -0,0 +1,217 @@
+  let mockLog: any;
+  const mockIsAnalyticsEnabled = vi.mocked(isAnalyticsEnabled);
+
+  beforeEach(async () => {
+    const analyticsModule = await import('../analytics');
+    mockLog = vi.mocked(analyticsModule.log);
+  });
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    mockIsAnalyticsEnabled.mockReturnValue(true);
+  });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)